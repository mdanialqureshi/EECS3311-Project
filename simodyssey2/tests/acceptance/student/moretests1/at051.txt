-- Test running out of fuel in test(1,1,1,1,30) mode
test(1,1,1,1,30)
move(S)
move(E)
move(E)