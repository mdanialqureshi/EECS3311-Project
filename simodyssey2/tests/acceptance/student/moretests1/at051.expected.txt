  state:0.0, ok
  Welcome! Try test(1,1,1,1,30)
->test(1,1,1,1,30)
  state:1.0, mode:test(1,1,1,1,30), ok
  Movement:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E---   *---   P---   ----   PP--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PY--   ----   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   P---   P*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PW--   ----   *---   ----   PW--   
->move(S)
  state:2.0, mode:test(1,1,1,1,30), ok
  Movement:
    [0,E]:[1,1,1]->[2,1,1]
    [3,P]:[1,5,2]->[5,4,1]
    [8,P]:[4,3,1]->[3,2,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *---   P---   ----   P---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    E---   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PY--   P---   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   ----   P*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PW--   ----   *---   P---   PW--   
->move(E)
  state:3.0, mode:test(1,1,1,1,30), ok
  Movement:
    [0,E]:[2,1,1]->[2,2,1]
    [8,P]:[3,2,1]->[4,1,2]
    [10,P]:[5,1,1]->[1,2,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *P--   P---   ----   P---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   E---   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PY--   ----   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    WP--   P*--   ----   P*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -W--   ----   *---   P---   PW--   
->move(E)
  state:4.0, mode:test(1,1,1,1,30), ok
  Explorer got lost in space - out of fuel at Sector:2:3
  The game has ended. You can start a new game.
  Movement:
    [0,E]:[2,2,1]->[2,3,2]
    [1,P]:[1,3,1]->[2,4,1]
    [2,P]:[1,5,1]->[5,1,1]
    [4,P]:[2,5,1]->[2,1,1]
    [6,P]:[3,4,1]->[4,4,3]
    [11,P]:[5,5,1]->[5,4,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *P--   ----   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    P---   ----   W---   P---   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PY--   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    WP--   P*--   ----   P*P-   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PW--   ----   *---   PP--   -W--   
